shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/condition/data/URIParameterData.java

```diff
@@ -17,17 +17,31 @@
 
 package org.apache.shenyu.plugin.base.condition.data;
 
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.shenyu.common.dto.ConditionData;
 import org.apache.shenyu.spi.Join;
 import org.springframework.web.server.ServerWebExchange;
 
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
 /**
  * The type URI parameter data.
  */
 @Join
 public class URIParameterData implements ParameterData {
-    
+
     @Override
     public String builder(final String paramName, final ServerWebExchange exchange) {
         return exchange.getRequest().getURI().getPath();
     }
+
+    @Override
+    public Boolean containsJudge(final ConditionData conditionData, final String realData) {
+        String regex = ",";
```

Comment:
URI have contains？ why not used match？

Reply:
Mainly to fix the IP contains problem, this upgrading for uri contains just fit the old test cast in PredicateJudgeFactoryTest and let uri can be contains, but i think this separator [,] will not be used